sirepo/pkcli/jupyterhublogin.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+"""CLI for jupyterhublogin
+
+:copyright: Copyright (c) 2021 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from __future__ import absolute_import, division, print_function
+import pykern.pkcli
+
+
+def create_user(email):
+    """Create a JupyterHubUser
+
+    Args:
+        email (str): Email of user to create.
+
+    Returns:
+        user_name (str): The user_name of the newly created user or existing user
+                        if email already exists.
+    """
+    import pyisemail
+    import sirepo.auth
+    import sirepo.server
+    import sirepo.sim_api.jupyterhublogin
+    import sirepo.template
+
+
+    assert pyisemail.is_email(email), \
+        f'invalid email={email}'
+    sirepo.server.init()
```

Comment:
This is very annoying. Perhaps pkcli needs a function that gets called from sirepo_console.py?

Reply:
Agreed. I'll think about it some https://github.com/radiasoft/sirepo/issues/3462